### DISCUSSION TITLE
The bot works!

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#228] [Team Posts]
If you saw a bunch of issues like this: https://github.com/reactjs/ja.reactjs.org/issues/41

Don't mind them -- they're just issues created by the translation bot. I managed to run the bot locally and update all three repos with the most recent update. For example: https://github.com/reactjs/ja.reactjs.org/pull/42

**Comment by alejandronanez** [comment]
This is amazing!

**Comment by tesseralis** [comment]
It's all due to @potato4d and the vuejs-jp team!

**Comment by potato4d** [comment]
@tesseralis 
Congratulations!

Currently, I’m doing implement new features of che-tsumi with re-fort.
If you have a feature request, feel free to create Issue!

**Comment by tesseralis** [comment]
I already started :) https://github.com/vuejs-jp/che-tsumi/issues/26

I think we're going to keep using a separate fork of it so we can move faster, but I'll try to add issues/PRs when I find something that might be useful generally!

**Comment by potato4d** [comment]
@tesseralis 
👌 

If you do not mind, can I also invite you to fork?
It seems that we can contribute to fork on the following functions which we are currently performing.

- Customize message
- When Bot can create issue, Add assigened label to issue
- Customize reviewer from environment var

**Comment by tesseralis** [comment]
Sure!
